src/Transaction.ts

```diff
@@ -96,28 +100,29 @@ export class Transaction {
 
   /**
    * Relays signed raw transactions.
-   * @param signedRawTx signed raw transaction
+   * @param signedTx signed ethereum transaction
    */
-  public relayTx (signedRawTx: string): Promise<string> {
+  public relayTx (signedTx: string): Promise<string> {
     const headers = new Headers({'Content-Type': 'application/json'})
     const options = {
       method: 'POST',
       headers,
-      body: JSON.stringify({rawTransaction: `0x${signedRawTx}`})
+      body: JSON.stringify({rawTransaction: `0x${signedTx}`})
     }
     return this._utils.fetchUrl<string>('relay', options)
   }
 
   /**
-   * Returns the latest block number.
+   * Returns the latest block number of the underlying blockchain.
    */
   public getBlockNumber (): Promise<number> {
     return this._utils.fetchUrl<number>('blocknumber')
   }
 
   /**
-   * Returns needed information for creating a transaction.
+   * Returns needed information for creating an ethereum transaction.
    * @param userAddress address of user creating the transaction
+   * @returns Information for creating an ethereum transaction for the given user address.
```

Comment:
What options? There comes an object with several values back, I think we should explain that somewhere.

Reply:
Maybe do it similar to the transaction option parameter. Say see tx info object, and then describe it somewhere else